examples/mundialis_example.py

```diff
@@ -43,6 +52,7 @@
     print(job.job_id)
     print(job.start_job())
     print(job.describe_job())
+    time.sleep(5)
```

Comment:
the client now provides `ImageCollectionClient.execute_batch`, which allows you to work with batch jobs in a much more compact way: it takes care of starting the job, polling the job status, waiting when job is finished, and downloading the result.

You should be able to write all the job stuff in this example as just:
```
datacube.execute_batch('/tmp/testfile')
```